server/tasks/stream_inference.py

```diff
@@ -71,7 +71,7 @@ async def run_streaming(
     task_time_limit=settings.task_time_limit_sec,
 )
 def streaming_parent_task(
-    self, agent: Agent, result_handler: ResultHandler, batch_size: int = 10
+    self, agent: Agent, result_handler: ResultHandler, batch_size: int = 1
```

Comment:
Why set the batch size to 1 here? is it just reverting another recent change in hopes that it's related to the instability?

Reply:
No, since we are sending "one message" from the producer in the agent now, which actually contains dozens of records usually, I dont want to pull too many at once, and overload LSE endpoints
Batch size of 1 here == tens of records to send to LSE 

before it was 10, which meant we pull 10 individual records. now we pull a batch

Reply:
ok yeah, so the actual batch size is the one set by adala for the consumer+producer in the kafka environment, which iirc defaults to 100

Reply:
yeah, it differs each batch though we dont actually wait for the full size